frontend/src/shared/cards.ts

```diff
@@ -0,0 +1,37 @@
+// Special card values
```

Comment:
The reason is that at the top level, it is not part of any TypeScript project, which the IDE was complaining about. If we ever switch the backend to using a Rollup build step as well, we could move it to the top, get rid of the symlinks and make all tsconfig files use the main folder as root with those files included, but currently this would completely mess up the backend build.